pkg/internal/testing/integration/assets/bin/.gitkeep

```diff
@@ -1 +0,0 @@
-This directory will be the home of some binaries which are downloaded with `pkg/framework/test/scripts/download-binaries`.
```

Comment:
I think you might want to keep that, for now, as the assets are currently still being downloaded to `.../pkg/internal/testing/integration/assets/bin/`, see: https://github.com/kubernetes-sigs/controller-runtime/blob/8c39906c77cdb574482c01fcd74a76b105b06522/hack/check-everything.sh#L86.

Reply:
yep, you're right, thanks. Updated...

Reply:
nit: If you want to remove references to 'download-binaries.sh' you might want to change the content of `.gitkeep` to something like:
```suggestion
This directory will be the home of some binaries which are downloaded with `make test` or `.../hack/check-everything.sh`.
```
🤷‍♂

Reply:
oh, right 🙈 thanks